transaction/extra/tests/verifier.rs

```diff
@@ -650,30 +639,35 @@ fn test_sci_tx_summary_verification() {
         &tx_summary,
         &tx_summary_unblinding_data,
         *bob.view_private_key(),
+        bob.change_subaddress(),
     )
     .unwrap();
     assert_eq!(
         &mlsag_signing_digest[..],
         &signing_data.mlsag_signing_digest[..]
     );
 
-    let balance_changes: Vec<_> = report
-        .balance_changes
-        .iter()
-        .map(|(x, y)| (x.clone(), *y))
-        .collect();
-    let mut expected = vec![
+    // TODO: fix this test
```

Comment:
So what is the TODO here? That we the "our address" output is not part of the totals ?